doc/mkdocs/docs/api/basic_json/at.md

```diff
@@ -25,15 +40,16 @@ const_reference at(const json_pointer& ptr) const;
 
 `key` (in)
 :   object key of the elements to access
-    
+
 `ptr` (in)
 :   JSON pointer to the desired element
-    
+
 ## Return value
 
 1. reference to the element at index `idx`
 2. reference to the element at key `key`
-3. reference to the element pointed to by `ptr`
+3. see 2
```

Comment:
Nitpicky, but I'd rather repeat `Logarithmic in the size of the container.` here. The level of indirection for that short sentence confuses me.